libraries/default.rb

```diff
@@ -12,7 +12,7 @@ module ProviderHelpers
     extend ::Ark::PlatformSpecificBuilders
 
     generates_archive_commands_for :seven_zip,
-      when_the: -> { node['platform_family'] == 'windows' },
+      when_the: -> { platform_family?('windows') },
```

Comment:
@tas50 I'm not sure why this is breaking but ChefSpec doesn't like this one bit.